### PR TITLE
fix(button-group): use status color for overflow button

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Button group ellipsis is always set to primary color.

Issue Number: CDE-1741

## What is the new behavior?
Button group ellipsis now use provided status color.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by kevinbuhmann — COMMENTED** [review]
This looks fine to me, but I would reword the PR title.

As a user, reading "remove ellipsis button color overrides" in the changelog wouldn't really be helpful. I would reword to "use status color for ellipsis button" or similar. Stating the observable/functional change is better than strictly what changed in the code.

**Review by dtsanevmw — APPROVED** [review]
